### DISCUSSION TITLE
Changes to support running LHCI in a closed network?

### DISCUSSION THREAD

**Discussion by EhhChris** [GoogleChrome/lighthouse-ci#635] [Ideas]
I currently run an older version of LHCI on a closed network, and there are a few things that we manually override/patch internally to get things running/working smoothly. This is mostly just replacing the links used for fonts/icons with local versions and such, but updating can be a bit of a pain.

Would the team maintaining this project be open to changes that make LHCI a little more out-of-the-box ready to run on closed networks? I've got a [fork with one commit](https://github.com/EhhChris/lighthouse-ci/commit/0352044750b852a56445d2bda71dac7aabac8369) right now that handles the majority of the problems on the lighthouse-ci side of things (I think), and would be happy to create a PR with it, but I've got very little experience with front-end and am not sure this is the most appropriate way of handling things- even if it is way better than what I do now.

**Comment by patrickhulce** [comment]
Yeah we'd be open to that! If you want to open a PR we can discuss the "right" way to go about some of that localization if it appears cumbersome :)
